backends/graphics/opengl/opengl-graphics.cpp

```diff
@@ -1294,16 +1294,15 @@ void OpenGLGraphicsManager::handleResizeImpl(const int width, const int height)
 	// possible and then scale it to the physical display size. This sounds
 	// bad but actually all recent chips should support full HD resolution
 	// anyway. Thus, it should not be a real issue for modern hardware.
-	if (   overlayWidth  > (uint)OpenGLContext.maxTextureSize
-	    || overlayHeight > (uint)OpenGLContext.maxTextureSize) {
+	if (overlayWidth > (uint)OpenGLContext.maxTextureSize || overlayHeight > (uint)OpenGLContext.maxTextureSize) {
```

Comment:
These changes are irrelevant, please remove all of them. If you want/need to do whitespace changes, those should go separately, but in this particular case, they are made on purpose, and obviously, it is just your code editor reformatting them, thus, please revert.